src/firmware/RC_firmware/RC_firmware.ino

```diff
@@ -0,0 +1,233 @@
+/* Firmware for Elsa
+   Test version
+   Author: Vincent Yuan, Modified: Nick Wu
+     Modified: James Asefa
+   Date: Oct 30, 2016
+*/
+
+#include <SoftwareSerial.h>
+#include <stdlib.h>
+#include <Servo.h>
+
+
+#define TRIM 8 // error margin for joysticks
+
+#define BAUD_RATE 115200
+#define BUFFER_SIZE 6
+// assign 128 to be not moving
+// will be used to hold robot in place while
+// reading from the buffer. Robot will be stopped initially.
```

Comment:
A quick comment here explaining what 0,128,256 map to in terms of physical movement (ie. 128 is stopped, 256 is full ahead, 0 is full back) would be helpful for people in the future.